src/Alpacka.Lib/Instances/MultiMC/MultiMCHandler.cs

```diff
@@ -12,6 +12,7 @@ public class MultiMCHandler : IInstanceHandler
     {
         private readonly Config _config = Config.Load();
         
+        public static readonly string CONFIG_FILE = "instance.cfg";
```

Comment:
maybe we should get the config file a different way.. and hide this field
i repeat thise lines anyway
```cs 
var multiMCInstancePath = Path.GetDirectoryName(instancePath);
var multiMCInstanceCfg = Path.Combine(multiMCInstancePath, CONFIG_FILE);
```